src/auth/auth.service.ts

```diff
@@ -12,7 +13,6 @@ export class AuthService {
 
   async validateUser(email: string, plainPassword: string): Promise<any> {
     const user = await this.usersService.findOneByEmail(email);
-
     if (user) {
       const isMatch = await bcrypt.compare(plainPassword, user.password);
```

Comment:
**suggestion**: Consider renaming `sub` to `userId` in the payload for better clarity.